platform/linux-generic/odp_random_openssl.c

```diff
@@ -32,6 +36,8 @@ int32_t _odp_random_openssl_data(uint8_t *buf, uint32_t len,
 		return (1 == rc) ? (int)len /*success*/: -1 /*failure*/;
 
 	case ODP_RANDOM_TRUE:
+		return _odp_random_true_data(buf, len);
```

Comment:
It is a bit inconvenient that this new random data is tied to OpenSSL and is not available if OpenSSL is not compiled in (which could easily be the case in ODP DPDK which can use DPDK crypto instead of OpenSSL crypto).

Maybe things could work so by default ODP_RANDOM_CRYPTO is provided by rdrand and ODP_RANDOM_TRUE by rdseed, and if OpenSSL is used, then ODP_RANDOM_CRYPTO comes from OpenSSL. Or maybe it should be configurable.

Reply:
> Maybe things could work so that by default ODP_RANDOM_CRYPTO is provided by rdrand

It appears that rdrand has as low bandwidth as rdseed in some CPUs so maybe it is not good for ODP_RANDOM_CRYPTO, at least without some configurability.

Reply:
Configure option added in v3.